src/pudl_archiver/zenodo/api_client.py

```diff
@@ -388,10 +388,8 @@ async def delete_file(self, file: DepositionFile):
             file: DepositionFile metadata pertaining to file to be deleted.
         """
         logger.info(f"DELETE file {file.filename} from {file.links.self}")
-        self._check_resp(
-            await self.session.delete(
-                file.links.self, params={"access_token": self.upload_key}
-            )
+        await self.session.delete(
```

Comment:
`_check_resp` assumes a JSON response, which this DELETE request doesn't have (it returns an empty `text/html`). Once we use the Depositor for this we can just pass `parse_json=False` and that should handle this.